tests/integration/testdata/sync/code/before/dotnet_function/src/HelloWorld/obj/HelloWorld.csproj.nuget.dgspec.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Sorry if I missed this my earlier pass, but it looks like we also don't need this `obj` folder under function source. I believe this is generated during build so we don't need to push this in git.